config/README.md

```diff
@@ -7,7 +7,12 @@ Copy this file to `~/.aidx/config.json` and update with your values:
   "appInsights": {
     "applicationId": "your-application-insights-id",
     "tenantId": "your-azure-tenant-id",
-    "endpoint": "https://api.applicationinsights.io/v1/apps"
+    "endpoint": "https://api.applicationinsights.io/v1/apps",
+    "subscriptionId": "your-azure-subscription-id",
```

Comment:
We don’t need these config change because appinsight's resource id is automatically detected, and azure data explorer is out of our scope.